doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -149,7 +149,9 @@ nRF5340 Audio
 nRF Machine Learning (Edge Impulse)
 -----------------------------------
 
-|no_changes_yet_note|
+* Updated:
+
+  * The machine learning models (:kconfig:option:`CONFIG_EDGE_IMPULSE_URI`) used by the application are now hosted by Nordic Semiconductor.
```

Comment:
```suggestion
* Updated the machine learning models (:kconfig:option:`CONFIG_EDGE_IMPULSE_URI`) used by the application so that they are now hosted by Nordic Semiconductor.
```

merged it to 1 line for now since it's just one item